### DISCUSSION TITLE
Adding Ckeditor fields to element index filtering options

### DISCUSSION THREAD

**Discussion by SETU-WEB** [craftcms/ckeditor#177] [Ideas] [answered]
This is one that has come up a lot without users who use filters wonder why their text field can be used to filter. Even if the only filters available where "where is empty" or "has a value" it would still be a good addition to element filtering setup

**Comment by brandonkelly** [comment, marked as answer]
Just released [craftcms/html-field](https://github.com/craftcms/html-field) 2.1.0 (Craft 4) and 3.0.0 (Craft 5) which adds a basic empty/not-empty condition rule to CKEditor (and Redactor) fields. Run `composer update` to get it.
